frontends/web/src/routes/account/account.tsx

```diff
@@ -47,29 +47,25 @@ interface LoadedAccountProps {
 }
 
 interface State {
-    accountSynced: boolean;
-    offlineMode?: boolean;
+    status?: accountApi.IStatus;
     transactions?: accountApi.ITransaction[];
     balance?: accountApi.IBalance;
     hasCard: boolean;
     exported: string;
     accountInfo?:accountApi. ISigningConfigurationList;
-    fatalError: boolean;
     syncedAddressesCount?: number;
 }
 
 type Props = LoadedAccountProps & AccountProps & TranslateProps;
 
 class Account extends Component<Props, State> {
     public readonly state: State = {
-        accountSynced: false,
-        offlineMode: undefined,
+        status: undefined,
```

Comment:
Nit: I'd prefer status not optional and initialize it here with the defaults, I guess `{ disabled: false, synced: false, fatalError: false, offline: false}`, WDYT?